Sources/CCFilter/ViewControllers/CCFilterViewController.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol CCFilterViewControllerDelegate: class {
```

Comment:
How will the changes in selection and current selection values be propagated to the "owner" of the filter?